UITests/BookmarkingTests.swift

```diff
@@ -0,0 +1,73 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import WebKit
+
+class BookmarkingTests: KIFTestCase, UITextFieldDelegate {
+    private var webRoot: String!
+
+    override func setUp() {
+        webRoot = SimplePageServer.start()
+    }
+
+    /**
+    * Tests basic page navigation with the URL bar.
+    */
+    func testBookmarkingUI() {
+        // Load a page
+        tester().tapViewWithAccessibilityLabel("URL")
+        let url1 = "\(webRoot)/?page=1"
+        tester().clearTextFromAndThenEnterText("\(url1)\n", intoViewWithAccessibilityLabel: "Address and Search")
+        tester().waitForWebViewElementWithAccessibilityLabel("Page 1")
+
+        // Bookmark it using the bookmark button
+        tester().tapViewWithAccessibilityLabel("Bookmark")
```

Comment:
Should we clear the profile or bookmarks DB before running this test? It'd be nice to prevent past failures from breaking future runs if they leave the DB in an unclean state.